src/components/button/index.njk

```diff
@@ -87,12 +87,6 @@ The `print` parameter can be set to `true` to render a print button. This will t
     patternlibExample({"path": "components/button/examples/button-print/index.njk"})
 }}
 
-### Exit
```

Comment:
Can we document using the `exit` param in the 'Ghost' example? so it's clear this should be used for save and sign out buttons